action.yml

```diff
@@ -32,6 +32,11 @@ inputs:
     description: "If truthy, the token will not be revoked when the current job is complete"
     required: false
     deprecationMessage: "'skip_token_revoke' is deprecated and will be removed in a future version. Use 'skip-token-revoke' instead."
+  # Make GitHub API consigurable to support GitHub Enterprise server usecases
+  # see https://github.com/actions/create-github-app-token/issues/77
+  github-api-url:
+    description: The API URL of the GitHub server.
```

Comment:
```suggestion
    description: The URL of the GitHub REST API.
```

Reply:
A nitpick: the https://api.github.com URL is the root for both the REST and the GraphQL APIs. On our GHES 3.9 instance `api.github.<website>` is a valid URL for this input instead of the legacy `github.<website>/api/v3` URL. Not sure if it changes the equation on exactly how to word this.

Reply:
We only use the REST API so that should be ok.